scripts/cddl_gen.py

```diff
@@ -1941,9 +1953,11 @@ def full_xcode(self, union_uint=None):
     def xcode(self):
         return self.full_xcode()
 
+    def xcoder_tuple(self):
+        return self.__class__._xcoder_tuple
```

Comment:
I don't understand the need for this method. Where is it being used? Isn't `xcoder_tuple` part of the module namespace and not this class's?

Reply:
Leftover from another iteration. Removed, thanks!